backend/codesirius/authentication/models/verification.py

```
@@ -6,6 +8,18 @@
 from django.utils.timezone import now, timedelta
 from rest_framework.exceptions import ValidationError
 
+from codesirius.kafa_producer import KafkaProducerSingleton
+
+logger = logging.getLogger(__name__)
+
+
+def delivery_report(err, msg):
```

Ruff (0.8.2)

Missing return type annotation for public function `delivery_report`

Add return type annotation: `None`

(ANN201)